lib/screens/v2/widget_instance/elevator_status_new.ex

```diff
@@ -89,23 +93,27 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatusNew do
   @max_callout_items 4
 
   @spec serialize(t()) :: Serialized.t()
-  def serialize(%__MODULE__{closures: closures, station_id: station_id}) do
+  def serialize(%__MODULE__{
+        closures: closures,
+        home_station_id: home_station_id,
+        relevant_station_ids: relevant_station_ids
+      }) do
     # Choose the first "scenario" that applies (ordered highest-priority first). Each function
     # returns a `Serialized` if it does apply or `nil` if it does not.
     Enum.find_value(
       [
-        &closed_here_without_nearby_backups/2,
-        &closed_elsewhere_without_in_station_backups/2,
-        &closed_elsewhere_with_in_station_backups/2,
-        &all_working_or_closed_with_nearby_backups/2
+        &closed_here_without_nearby_backups/3,
+        &closed_elsewhere_without_in_station_backups/3,
+        &closed_elsewhere_with_in_station_backups/3,
+        &all_working_or_closed_with_nearby_backups/3
       ],
-      & &1.(closures, station_id)
+      & &1.(closures, home_station_id, relevant_station_ids)
     )
```

Comment:
I really like this pattern, but was starting to think it was a little strange that none of the functions actually "need" an arity of 3. I tried to see if there was another way to do this, and was thinking using anonymous functions like 
```
Enum.find_value(
  [
    fn -> closed_here_without_nearby_backups(closures, home_station_id) end,
    fn -> closed_elsewhere_without_in_station_backups(closures, relevant_station_ids) end,
    fn -> closed_elsewhere_with_in_station_backups(closures) end,
    fn -> all_working_or_closed_with_nearby_backups(closures) end
  ],
  & &1.()
)
```

I don't actually think I have strong feelings that one way is better than the other, but I'm sharing because I typed it up and I just think Elixir is neat 😄

Reply:
No you're cooking, this is better I think, especially since #2791 adds yet _another_ argument that only some of the cases care about. Will change this.